components/css/partials/_accordion.scss

```diff
@@ -32,5 +41,29 @@
 				content:"\f0fe";
 			}
 		}
+		@each $color,$pair in $colorPairs-accessible{
+			&-#{$color}{
+				background-color:nth($pair,1);
+				color:nth($pair,2);
+			}
+		}
+	}
+	&_list{
+		margin:0;
+		padding:0 20px;
+		list-style-type:none;
+	}
+	&_item{
+		border-top:solid 1px #000;
+		margin:0;
+		padding:15px 0 !important;
+		&:first-child{
+			border-top:none;
+		}
+		@each $color,$pair in $colorPairs-accessible{
+			&-#{$color}{
+				border-top-color:nth($pair,1);
+			}
+		}
 	}
 }
```

Comment:
Overall: do we want to consider an alternating color pallete for the headings? Just recently I applied some styles to an accordion for a relatively long, unbroken FAQ section and Garrett had me alternate the heading colors to break up the wall of color it created when all the accordions were closed. 

**Screenshot**

![screen shot 2018-01-24 at 2 13 32 pm](https://user-images.githubusercontent.com/5982996/35352113-e1a0a9b0-0110-11e8-8a0a-c962427feeba.png)

Reply:
I know this is coming in late, but the black on gray background seems harder to read and blends together a bit. It may be better to make the gray alternating colors lighter? #f2f2f2 and #d1d1d1?

Reply:
@jobokai Let me know what you think about the latest striped versions. The screenshots above will use one of the gray variations instead.